.env.example

```diff
@@ -3,6 +3,7 @@ ANNOUNCED_ICO_START_DATE="2017-09-15T13:37:19+00:00"
 COMMITMENT_CONTRACT_ADDRESS=0x40a30b4fbf7ef186515c21ac11abcbbe99cdba5e
 RPC_PROVIDER=http://localhost:8545
 GA_ID=UA-00000000-0
+PIXEL_ID=00000000
```

Comment:
sorry for doing it ;d but maybe name it `FB_PIXEL_ID` - if I would see PIXEL_ID somewhere I wouldn't have any idea what's it referening to